style.css

```diff
@@ -0,0 +1,96 @@
+/* general style  */
+* {
+  padding: 0;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
+}
+
+li {
+  list-style: none;
+}
+
+/* navbar style */
+.nav-bar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.7rem 1rem;
+  color: rgb(13, 97, 253);
+  margin-bottom: 10vh;
+}
+
+.nav-bar i {
+  font-size: 1.2rem;
+  transition: 0.5s;
+  cursor: pointer;
+}
+
+/* header section style  */
+.header {
+  width: 100%;
+  height: 100vh;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 1rem;
+  color: rgb(4, 4, 59);
+  background-image: url(./images/shapes.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  margin-top: 4rem;
+  background-color: #fff;
+}
+
+.container {
+  margin-top: 5rem;
+}
+
+.header h1 {
+  font-size: 50px;
+  margin-bottom: 1.5rem;
+  color: #091e42;
+  line-height: 52px;
+}
```

Comment:
- [ ] Kindly make sure to stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. This includes all typography, colors, margins, padding etc. Kindly change the `color: #091242;` to `color: #172B4D;` which is provided in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [ ] Also, kindly make sure that this heading has a `font-weight: 700;` which is provided in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).